examples/covariance/plot_covariance_estimation.py

```diff
@@ -66,7 +66,7 @@
 
 # #############################################################################
 # Compute the likelihood on test data
-
+#
```

Comment:
Some of these single `#` does not change the formatting. In this example, [PR](https://101468-843222-gh.circle-artifacts.com/0/doc/auto_examples/covariance/plot_covariance_estimation.html) and [master](https://scikit-learn.org/dev/auto_examples/covariance/plot_covariance_estimation.html)

This is most likely because the `#####` is not connected.